webdriver-spec.html

```diff
@@ -4624,636 +4646,562 @@
-          <li>Releasing the Shift key</li>
-        </ul>
-        Alternatively, it can be achieved by:
-        <ul>
-          <li>Holding down the Shift key</li>
-          <li>Pressing the letter 'a'</li>
-          <li>Releasing the Shift key</li>
-          <li>Holding down the Shift key</li>
-          <li>Pressing the letter 'b'</li>
-          <li>Releasing the Shift key</li>
-        </ul>
-        <p>Or by simply turning on the CAPS LOCK first.</p>
-        <p>
-        The implementation MAY use the following algorithm to generate the events.
-        If the implementation is using a different algorithm, it MUST adhere to the
-        requirements listed below.
-        <p>For each key, <var>key</var> in <var>value</var>, do
-        <ol>
-          <li>If <var>key</var> is a lower-case symbol:
+       <p>The remote end will dispatch the <code><a>pointerDown</a></code> action in the first tick.
+        In the second tick, since source 1 declares a <code><a>pause</a></code> of 5 seconds,
+        the remote end will dispatch the <code><a>pointerUp</a></code> event for source 2,
+        and will wait 5 seconds before moving on to executing the third tick.
+
+       <p>In the event that one tick contains multiple <code><a>pause</a></code> durations,
+        the remote end will wait the maximum duration
+        before moving on to executing the next tick.
+
+       <p>As noted before, <code><a>pause</a></code> can be used to signify inaction during a tick.
+        If <code><a>pause</a></code> is declared without a time period,
+        then the input source will not have any actions executed in the containing tick.
+        As an example:
+
+       <p><img src=graphics/note3actions.svg alt="">
+
+       <p>During tick 2, source 1 will have its pointerMove action dispatched,
+        while source 2 will do nothing.
+    </aside>
+
+    <p>
+
+    <p>The following <dfn>table of action commands</dfn> enumerates all supported
+    <a>commands</a>
+    <table>
+      <tr>
+        <th>Action Command
+        <th>Input Type
+      </tr>
+
+      <tr>
+        <td><a>keyDown</a>
+        <td><a>Keyboard</a>
+      </tr>
+
+      <tr>
+        <td><a>keyUp</a>
+        <td><a>Keyboard</a>
+      </tr>
+
+      <tr>
+        <td><a>pointerMove</a>
+        <td><a>Pointer</a>
+      </tr>
+
+      <tr>
+        <td><a>pointerDown</a>
+        <td><a>Pointer</a>
+      </tr>
+
+      <tr>
+        <td><a>pointerUp</a>
+        <td><a>Pointer</a>
+      </tr>
+
+      <tr>
+        <td><a>pointerCancel</a>
+        <td><a>Pointer</a>
+      </tr>
+      <tr>
+        <td><a>pause</a></td>
+        <td><a>General</a></td>
+      </tr>
+    </table>
+
+    <p><table class="simple jsoncommand">
+      <tr>
+      <th>HTTP Method</th>
+        <th>Path Template</th>
+      </tr>
+      <tr>
+        <td>POST</td>
+        <td>/session/{<var>session id</var>}/actions</td>
+      </tr>
+    </table>
+    <p>The <a>remote end steps</a> are:
```

Comment:
First, check if the window is still open and whether there are any open user prompts.